src/controls/NumberControl.test.tsx

```diff
@@ -147,17 +134,3 @@ it ("shows units in text control if set in UI schema", async () => {
   })
   expect(await screen.findByText("°F")).not.toBeNull()
 })
-
-it ("shows units in tooltip if set in UI schema", async () => {
-  render({
-    schema: numberHumiditySchema,
-    uischema: numberPercentageUISchema,
-  })
-  // we are using a schema that doesn't have the text input,
-  // so we can be sure the units are showing up in the tooltip
-  expect(screen.queryByRole("spinbutton")).toBeNull()
-  const slider = screen.getByRole("slider")
-  expect(screen.queryByText("0%")).toBeNull()
-  await userEvent.hover(slider)
-  expect(await screen.findByText("0%")).not.toBeNull()
-})
```

Comment:
Test no longer applies since we aren't conditionally adding the text input with the slider anymore. There is another test that checks the tooltip in `NumericSliderControl.test.tsx`.